src/Microsoft.DotNet.XHarness.Android/Execution/AdbProcessManager.cs

```diff
@@ -90,15 +89,15 @@ public ProcessExecutionResults Run(string adbExePath, string arguments, TimeSpan
         p.Close();
 
         lock (standardOut)
-            lock (standardErr)
+        lock (standardErr)
```

Comment:
What are those locks needed for?

Reply:
I guess they make sure we ended writing the output before returning from here.
I guess this was some auto-format change of this file

Reply:
I looked at history of this file and at the beginning they were there to solve some concurrency issues but in the meantime the code was changed a bit and now the process is closed before we even get to this section. I'm pretty sure they are not needed any more but who knows. Process API has a lot of strange edge cases.

Reply:
I agree, seems like the process should be long dead when we get here

Reply:
Only if the Kill failed..